redaxo/src/addons/structure/plugins/history/lib/history_login.php

```diff
@@ -29,7 +29,7 @@ public function checkTempSession($historyLogin, $historySession, $historyValidti
     /**
      * @return null|string
      */
-    public static function createSessionKey($login, $session, $validtime)
+    public static function createSessionKey(#[\SensitiveParameter] $login, $session, $validtime)
     {
```

Comment:
Die Notation ist so nicht PHP7-kompatibel, da der Kommentar bis zum Zeilenende geht.
Das Attribut (bzw. der Kommentar) muss also in einer eigenen Zeile stehen.